internal/store/postgres/invitation.go

```diff
@@ -26,7 +26,7 @@ func (from Invitation) transformToInvitation() (invitation.Invitation, error) {
 		}
 	}
 	var groupIDs []string
-	if val, ok := unmarshalledMetadata["group_ids"]; ok {
+	if val, ok := unmarshalledMetadata["group_ids"]; ok && (val != nil) {
```

Comment:
Do you think this check was unnecessary? I don't remember the case, but I saw `val` being null in sometimes.

Reply:
I created a user invitatioon just for an org, (without any group_ids) and listed the pending invitations. There was an error for trying to unmarshall the null value in the server logs

Reply:
Exact error `interface conversion: interface {} is nil, not []interface {}`

Reply:
Cool, make sense